statemachine/src/test/java/com/amazonaws/stepfunctions/cloudformation/statemachine/UpdateHandlerTest.java

```diff
@@ -183,6 +184,19 @@ public void testDefinitionFromS3() throws Exception {
         assertThat(response.getResourceModel().getDefinitionString()).isEqualTo("{}");
     }
 
+    @Test
+    public void testWhenArnMissing_throwsStateMachineDoesNotExist() {
```

Comment:
This test is repetitive as the one in requestHandler. They serve the same purpose so I think we can save the one under ResourceHandlerTest suit

Reply:
There's no test suite for the base ResourceHandler class, only test suites for the actual handlers. I'm in favour of leaving them in the individual handler test suites since only some of the handlers have this constraint.